Packs/Core/ReleaseNotes/3_0_70.md

```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### New: Scheduled task created with HTTP or FTP reference
+
+- New: This playbook is designed to handle the alert "Scheduled task created with HTTP or FTP reference".
+
+The playbook executes the following stages:
+
+Investigation:
+During the alert investigation, the playbook will perform the following:
+- Checks the IP and the URL reputation.
+- Checks the CGO process signature.
+- Search for related XDR agent alerts to determine if the creation of the scheduled task is part of an attack pattern.
```

Comment:
Searches for related XDR agent alerts to determine if the creation of the scheduled task is part of an attack pattern.
Comment